src/pyfmi/fmi3.pyx

```diff
@@ -425,6 +422,11 @@ cdef class FMUModelME3(FMUModelBase3):
         else:
             raise InvalidVersionException('The FMU could not be loaded. This class only supports FMI 3.0 for Model Exchange.')
 
+    def get_identifier(self):
+        if not self._modelId:
```

Comment:
Note that the docstring is automatically fetched from the parent:
```
In [4]: fmu.get_identifier?
Signature: fmu.get_identifier()
Docstring:
Return the model identifier, name of binary model file and prefix in
the C-function names of the model.
Type:      method
```

Reply:
Neat, didn't know that!